components/empty-state/empty-state-illustrated-button.js

```diff
@@ -44,11 +44,9 @@ class EmptyStateIllustratedButton extends RtlMixin(LitElement) {
 
 	render() {
 		return html`
-		<div>
-                ${this.illustration ? html`${runAsync(this.illustration, () => getIllustration(this.illustration), {
+			${this.illustration ? html`${runAsync(this.illustration, () => getIllustration(this.illustration), {
```

Comment:
Nit: I might be inclined to name this `illustrationName`/`illustration-name` for clarity... so consumers don't get the impression that they can stuff an `svg` in there.

Reply:
Yep, good call